modules/tensor_mechanics/include/materials/ADComputeFiniteStrainElasticStress.h

```diff
@@ -32,8 +33,12 @@ class ADComputeFiniteStrainElasticStress : public ADComputeStressBase, public Gu
   const std::string _elasticity_tensor_name;
   /// Elasticity tensor material property
   const ADMaterialProperty<RankFourTensor> & _elasticity_tensor;
-
   const ADMaterialProperty<RankTwoTensor> & _strain_increment;
+  /// Rotation up to step "n" to compute non-isotropic elasticity tensor
```

Comment:
This first one should be the rotation up to the end of the current step, right? We can't call both of these times "n".